src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
-package core.basesyntax;
-
-public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
-    }
-}
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private static final Random randomIndex = new Random();

```

Comment:
Don’t  use static for instances 
```suggestion
    private final Random random = new Random();

```